# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/wasi-libcxx/hotfix-libcxx.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./libcxx/src/CMakeLists.txt.vanilla	2025-03-05 23:29:53.520370605 +0100
+++ ./libcxx/src/CMakeLists.txt	2025-03-05 23:29:59.337312976 +0100
@@ -4,7 +4,6 @@
   any.cpp
   bind.cpp
   call_once.cpp
-  charconv.cpp
   chrono.cpp
   error_category.cpp
   exception.cpp
@@ -28,8 +28,6 @@
   include/ryu/digit_table.h
   include/ryu/f2s.h
   include/ryu/ryu.h
-  include/to_chars_floating_point.h
-  include/from_chars_floating_point.h
   memory.cpp
   memory_resource.cpp
   new_handler.cpp
